participants.js

```diff
@@ -0,0 +1,79 @@
+// Define the W3C's group ID
+const GROUP_ID = '83726';
```

Comment:
FWIW, I don't think we need to wait for the aria.js work to complete since this file is loaded separately. Let's try to make a PR on aria-common.